[
  {
    "url": "https://api.github.com/repos/lodash/lodash/issues/comments/384197381",
    "html_url": "https://github.com/lodash/lodash/issues/3753#issuecomment-384197381",
    "issue_url": "https://api.github.com/repos/lodash/lodash/issues/3753",
    "id": 384197381,
    "node_id": "MDEyOklzc3VlQ29tbWVudDM4NDE5NzM4MQ==",
    "user": {
      "login": "amullier",
      "id": 22471855,
      "node_id": "MDQ6VXNlcjIyNDcxODU1",
      "avatar_url": "https://avatars.githubusercontent.com/u/22471855?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/amullier",
      "html_url": "https://github.com/amullier",
      "followers_url": "https://api.github.com/users/amullier/followers",
      "following_url": "https://api.github.com/users/amullier/following{/other_user}",
      "gists_url": "https://api.github.com/users/amullier/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/amullier/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/amullier/subscriptions",
      "organizations_url": "https://api.github.com/users/amullier/orgs",
      "repos_url": "https://api.github.com/users/amullier/repos",
      "events_url": "https://api.github.com/users/amullier/events{/privacy}",
      "received_events_url": "https://api.github.com/users/amullier/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2018-04-25T08:05:31Z",
    "updated_at": "2018-04-25T08:05:31Z",
    "author_association": "NONE",
    "body": "Hello ! What was the solution please ?",
    "reactions": {
      "url": "https://api.github.com/repos/lodash/lodash/issues/comments/384197381/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/lodash/lodash/issues/comments/384198052",
    "html_url": "https://github.com/lodash/lodash/issues/3753#issuecomment-384198052",
    "issue_url": "https://api.github.com/repos/lodash/lodash/issues/3753",
    "id": 384198052,
    "node_id": "MDEyOklzc3VlQ29tbWVudDM4NDE5ODA1Mg==",
    "user": {
      "login": "jdalton",
      "id": 4303,
      "node_id": "MDQ6VXNlcjQzMDM=",
      "avatar_url": "https://avatars.githubusercontent.com/u/4303?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/jdalton",
      "html_url": "https://github.com/jdalton",
      "followers_url": "https://api.github.com/users/jdalton/followers",
      "following_url": "https://api.github.com/users/jdalton/following{/other_user}",
      "gists_url": "https://api.github.com/users/jdalton/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/jdalton/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/jdalton/subscriptions",
      "organizations_url": "https://api.github.com/users/jdalton/orgs",
      "repos_url": "https://api.github.com/users/jdalton/repos",
      "events_url": "https://api.github.com/users/jdalton/events{/privacy}",
      "received_events_url": "https://api.github.com/users/jdalton/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2018-04-25T08:07:56Z",
    "updated_at": "2018-04-25T08:07:56Z",
    "author_association": "MEMBER",
    "body": "The file exists so I'm guessing the error is on your end and not ours.",
    "reactions": {
      "url": "https://api.github.com/repos/lodash/lodash/issues/comments/384198052/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  }
]